contract_manager/scripts/common.ts

```diff
@@ -181,3 +181,37 @@ export function getSelectedChains(argv: {
   }
   return selectedChains;
 }
+
+/**
+ * Finds the entropy contract for a given EVM chain.
+ * @param chain The EVM chain to find the entropy contract for.
+ * @returns The entropy contract for the given EVM chain.
+ * @throws an error if the entropy contract is not found for the given EVM chain.
```

Comment:
nit: According to [jsdoc](https://jsdoc.app/tags-throws), The first expression after `@throws` should be type of error or something like `Error`. Currently syntax highlighting is a bit weird:
<img width="1088" alt="image" src="https://github.com/pyth-network/pyth-crosschain/assets/6235952/1c29517a-8cd5-4cc3-9464-01a5c44d1d5a">